views/layout/basic-card.html

```diff
@@ -0,0 +1,7 @@
+{% from "components/basic-card.html" import basicCard %}
```

Comment:
I gave this a shot. The routes are not setup yet, so I wasn't sure what exactly to do here. I looked at a portion of the `products` `view` that used `{% set %}` as a [reference](https://github.ibm.com/Watson/wox-ui/blob/develop/views/products/index.html#L12), but it breaks locally so I'm sure it's not right.